src/utils/index.js

```diff
@@ -9,6 +9,7 @@ module.exports.checkHeadsetConnected = require('./checkHeadsetConnected');
 module.exports.debug = require('./debug');
 module.exports.entity = require('./entity');
 module.exports.forceCanvasResizeSafariMobile = require('./forceCanvasResizeSafariMobile');
+module.exports.functions = require('./functions');
```

Comment:
I don't think functions is a very good name. All modules export functions. I would just name the file `bind.js` an do `require('./bind');`